changelog/10940.improvement.rst

```diff
@@ -0,0 +1 @@
+Added :func:`TerminalReporter.wrap_write() <pytest.TerminalReporter.wrap_write>`, which improves verbose output by performing text wrapping on skip and xfail reasons while leaving a clear margin for progress output.
```

Comment:
Perhaps it would be better to swap the statements, given the last one is more important to users than the addition of the internal method:

```suggestion
Improved verbose output (``-vv``) of ``skip`` and ``xfail`` reasons by performing text wrapping while leaving a clear margin for progress output.

Added :func:`TerminalReporter.wrap_write() <pytest.TerminalReporter.wrap_write>` as a helper for that.
```